ENIGMAsystem/SHELL/Graphics_Systems/OpenGL3/GL3shader.cpp

```diff
@@ -642,7 +642,7 @@ void glsl_program_detach(int id, int sid)
 void glsl_program_set(int id)
 {
   if (enigma::bound_shader != id){
-    oglmgr->ShaderFunc();
```

Comment:
Each of the `oglmgr` functions I removed was just for flushing the batch, that's all they did, and some of them didn't actually do that. They were just left empty. I don't remember if that was one of them. The reason I am so interested in doing away with these and creating `GSstate.h`/`GSstate.cpp` in a future pr is because I don't feel we should have a lot of graphics code in `Bridges/General/` because that makes no sense. I also feel that the logic should be deduplicated, see my other comments below about my plans. Finally, I feel that the user should be given more control over the whole thing (hence the niche & novel features in this pull request).